example_specs/workflows/stardist/stardist_example.yaml

```diff
@@ -0,0 +1,70 @@
+name: StarDist Example Workflow
+description: A workflow for running stardist
+format_version: 0.2.3
+type: workflow
+
+inputs:
+- name: raw
+  type: tensor
+  description: image with star-convex objects
+
+outputs:
+- name: labels
+  type: tensor
+- name: coord
+  type: tensor
+- name: points
+  type: tensor
+- name: prob
+  type: tensor
+
+kwargs:
+- name: diameter
+  type: float
+  default: 2.3
+
+steps:
+- op: zero_mean_unit_variance
+- op: model_inference
+  kwargs:
+    rdf_source: fearless-crab
+    preprocessing: false # disable the preprocessing
+    postprocessing: false # disable the postprocessing
+- op: stardist_postprocessing
+  kwargs:
+    diameter: ${{ kwargs.diameter }}
+
+test_steps:
+- id: test_tensors
+  op: load_tensors
+  outputs:
+    - raw
+    - labels
+    - coord
+    - points
+    - prob
+  kwargs:
+    sources:
+    - raw.npy
+    - labels.npy
+    - coord.npy
+    - points.npy
+    - prob.npy
+- id: workflow
+  op: run_workflow
+  inputs: [test_tensors.outputs.raw]
+  outputs:
+    - labels
+    - coord
+    - points
+    - prob
+  kwargs:
+    rdf_source: ${{ self.rdf_source }}
```

Comment:
Question: The kwargs can also accept runtime outputs, something like `${{ stepx.outputs.labels }}`, right?

Reply:
Yes, why not!